src/modules/api/user/link/link.controller.ts

```diff
@@ -1,7 +1,12 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Post, Body, Param } from '@nestjs/common';
 import { LinkService } from './link.service';
 
 @Controller('users/:userId/links')
 export class LinkController {
-  constructor(private linkService: LinkService) {}
+  constructor(private linkService: LinkService) { };
+  @Post()
+  async create(@Body() body, @Param() params) {
+     const result = await this.linkService.create(body,params.userId);
```

Comment:
don't send the body directly. Please destructure the data and send individually.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#:~:text=The%20destructuring%20assignment%20syntax%20is%20a%20JavaScript%20expression,distinct%20variables.%20JavaScript%20Demo%3A%20Expressions%20-%20Destructuring%20assignment